rend3-framework/src/assets.rs

```diff
@@ -64,7 +66,16 @@ impl AssetLoader {
         path.get_path(&self.base)
     }
 
-    #[cfg(all(not(target_arch = "wasm32"), not(target_os = "android")))]
+    #[cfg(all(not(target_arch = "wasm32"), not(target_os = "android"), not(target_os = "wasi")))]
```

Comment:
Why is this function fully duplicated?

Reply:
The initial implementation on `wasm32-wasi` was different and I failed to recognize that I undid all changes to the native method. I'm no longer sure why I really did changes in the first place, mainly I had the paths rooted at the root directory.